server/container/guice/jpa-guice/src/main/java/org/apache/james/modules/mailbox/JPAMailboxModule.java

```diff
@@ -85,7 +87,9 @@ protected void configure() {
         bind(SimpleGroupMembershipResolver.class).in(Scopes.SINGLETON);
         bind(UnionMailboxACLResolver.class).in(Scopes.SINGLETON);
         bind(DefaultMessageId.Factory.class).in(Scopes.SINGLETON);
+        bind(JPAMessageIdManager.class).in(Scopes.SINGLETON);
 
+        bind(MessageIdManager.class).to(JPAMessageIdManager.class);
```

Comment:
There is no things like a JPAMessageIdManager ....

You don't need it, remove it. It's a lie.